views/dashboard/index.html

```diff
@@ -1,13 +1,25 @@
 <div class="search-keyword">
-  <form class="form" action="{{ urlfor `ScraperController.Create` }}" method="post">
+  <form class="form" action="{{ urlfor `ScraperController.TextSearch` }}" method="post">
+    {{ .XSRFData }}
+
     <div class="form__group">
       <input class="form__input" type="text" name="keyword" placeholder="Search keywords here..." required>
 
       <button type="submit" class="button button--icon">
         <span class="material-icon">search</span>
       </button>
+
+      <button id="upload-file-button" type="button" class="button button--icon">
```

Comment:
There is also the attribute `form` to link a button to a form. See https://www.w3schools.com/tags/att_button_form.asp

Reply:
I mapped form to the outside element naming with class name inside this attribute `data-action-control` for binding click event the can control `input[type="file"]` to open the browser picker 🤣